safer-ffi-gen/Cargo.toml

```
@@ -3,12 +3,19 @@ name = "safer-ffi-gen"
 version = "0.1.0"
 edition = "2021"
 
+[features]
+default = []
+tokio = ["dep:tokio", "dep:once_cell"]
+
 [dependencies]
 safer-ffi-gen-macro = { path = "../safer-ffi-gen-macro/"}
-safer-ffi = { version = "0.0.10", features = ["proc_macros", "out-refs"] }
+safer-ffi = { version = "0.0.10", features = ["proc_macros", "out-refs", "headers"] }
```

```suggestion
safer-ffi = { version = "0.0.10", features = ["proc_macros", "out-refs"] }
```

Untested, but here's the diff if you want to remove it.